fix use of renamed VTK method InsertNextTupleValue, is now InsertNextTypedTuple

author: Kenneth Hoste (HPC-UGent)
--- ANTs-2.3.1/Examples/antsSurf.cxx.orig	2018-10-12 14:23:41.927287114 +0200
+++ ANTs-2.3.1/Examples/antsSurf.cxx	2018-10-12 14:23:53.477426876 +0200
@@ -478,7 +478,7 @@
     currentColor[2] = static_cast<unsigned char>( currentBlue  * 255.0 );
     currentColor[3] = static_cast<unsigned char>( currentAlpha * 255.0 );
 
-    colors->InsertNextTupleValue( currentColor );
+    colors->InsertNextTypedTuple( currentColor );
     }
   vtkMesh->GetPointData()->SetScalars( colors );
 
